spec/requests/admin/users_controller_spec.rb

```diff
@@ -1049,4 +1049,18 @@ def find_logs(action)
     end
   end
 
+  describe '#sso_record' do
+    fab!(:sso_user) { Fabricate(:user) }
```

Comment:
minor but we already have `fab!(:user) { Fabricate(:user) }` at the top of this file so we can just use that.